packages/mask/src/plugins/Pets/types.ts

```diff
@@ -0,0 +1,34 @@
+import type { ERC721TokenInfo } from '@masknet/web3-shared-evm'
+
+export interface PetsDialogEvent {
+    open: boolean
+}
+
+export interface EssayRSSNode {
+    address: string
+    signature: string
+    essay: PetMetaDB
+}
+
+export interface PetMetaDB {
+    userId: string
+    tokenId: string
+    contract: string
+    word: string
+    image: string
+}
+
+export interface User {
+    userId: string
+    address: string
+}
+
+export interface OwnerERC721TokenInfo extends ERC721TokenInfo {
```

Comment:
What do you need is `ERC721TokenDetailed` here?

Reply:
ERC721TokenInfo not contain tokenId